src/ShopBundle/pkgExtranet/objects/db/TShopDataExtranetUser.class.php

```diff
@@ -16,6 +16,8 @@ class TShopDataExtranetUser extends TShopDataExtranetUserAutoParent
 {
     const COOKIE_NAME_HISTORY = 'shopuserarticleviewhistory';
     const COOKIE_NAME_NOTICELIST = 'shopuserarticlenoticelist';
+    private const MAX_NOTICE_LIST_COOKIE_LENGTH = 25; // this should stay under the critical 4kb string size
```

Comment:
This is a little flimsy in my opinion. Isn't there a possibility to calculate the actual size of the resulting cookie and limit it that way? It would remove the guess work involved in limiting the number of items of which we do not know the size in advance.

Reply:
There are other flimsy aspects here as well. For example the 4kb cookie size limit is most likely web server-dependant.
So it could be more or less.

In that context this limit is a "best guess" and an easy solution.

Reply:
You can't add more than ~25 elements yet already. So it doesn't get worse. So a limit would be o.k. with me, if the users gets a message, that he reached such limit and needs to login to add more (which is good for the shop owner).

Reply:
There is no reasonable possibility for an error message as there is no general message spot that could be targeted.

Reply:
ok, so if we fix the BC break I think we can go with the guessed limit. It is not pretty but that's how it is, I guess.